types/testingutils/keymanager.go

```diff
@@ -191,12 +192,25 @@ func (km *testingKeyManager) Decrypt(pk *rsa.PublicKey, cipher []byte) ([]byte,
 
 // Encrypt given a rsa pubkey and data returns an PKCS1v15 e
 func (km *testingKeyManager) Encrypt(pk *rsa.PublicKey, data []byte) ([]byte, error) {
-	panic("implement")
+	return FakeEncryption(data), nil
 }
 
 // SignDKGOutput signs output according to the SIP https://docs.google.com/document/d/1TRVUHjFyxINWW2H9FYLNL2pQoLy6gmvaI62KL_4cREQ/edit
 func (km *testingKeyManager) SignDKGOutput(output types.Root, address common.Address) (types.Signature, error) {
-	panic("implemet")
+	root, err := output.GetRoot()
+	if err != nil {
+		return nil, err
+	}
+	sig := FakeEcdsaSign(root, address[:]) // GLNOTE: Use fake so that we have deterministic message
+	return sig, nil
+}
+
+func (km *testingKeyManager) VerifyDKGOutput(signature types.Signature, output types.Root, address common.Address) bool {
```

Comment:
verify should not be part of the key manager struct

Reply:
You're right. I put it here so that we can see the (mock) sign and verify side by side.